src/main/resources/public/raml/resourceTypes/rule-test.raml

```diff
@@ -0,0 +1,30 @@
+description: Using the root endpoint one can check if rule tests are enabled in the EI instance.
```

Comment:
I think you misunderstood my comment. This could have been the way it was before, with the endpoint changed. xD